studio/src/windowManagement/PreferencesPopover.js

```diff
@@ -57,7 +57,7 @@ export class PreferencesPopover extends Popover {
 	 * @param {HTMLElement} buttonEl
 	 * @param {import("../../../src/mod.js").UuidString} contentWindowUuid
 	 */
-	initialize(preferencesManager, preferenceIds, buttonEl, contentWindowUuid) {
+	initialize = (preferencesManager, preferenceIds, buttonEl, contentWindowUuid) => {
```

Comment:
You can use regular method syntax now that this doesn't need to override anything anymore. Same for for the EntryPointPopover.